tests/integration/defs/triton_server/test_list_parser.py

```
@@ -103,7 +103,22 @@ def parse_test_name(s):
+                    sm_version = int(sm_match.group(1))
+                    # Get current SM version
+                    try:
+                        from ..conftest import get_sm_version
+                        current_sm = get_sm_version()
+                        # If SM versions match, replace with test_prefix
+                        if sm_version == current_sm:
+                            s = s.replace(f'sm{sm_version}',
+                                          test_prefix.split('-')[0])
+                    except Exception:
+                        # If can't get SM version, skip SM-based filtering
+                        pass
```

Ruff (0.14.3)

`try`-`except`-`pass` detected, consider logging the exception

(S110)